func_sig_registry/registry/models.py

```diff
@@ -141,3 +145,99 @@ def get_hex_display(self):
 
     def get_bytes_display(self):
         return force_text(self.bytes4_signature)
+
+
+class EventSignature(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True, db_index=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    text_signature = models.TextField(unique=True,
+                                      validators=[
+                                          MinLengthValidator(3),
```

Comment:
it *might* be prudent for us to set an upper bound on this, even though there theoretically doesn't have to be one.  

- pick a number from your gut 1024
- something like 3x the largest event signature you can find in the wild
- look at solidity or vyper and see if either of them have strict upper bounds that they enforce in their compiler.